README.adoc

```diff
@@ -219,24 +219,24 @@ In the `start` directory, run the following command to package the `system` micr
 mvn -pl system package
 ```
 
-The `mvn package` command will compile, verify, and build the project. The resulting compiled code will be packaged into a `war` web archive which can be 
-found at `system/target/system.war`. The archive contains all the configuration files, as well as the application needed to run the microservice on an 
-Open Liberty server, and is now ready to be injected into a Docker container for deployment.
+The `mvn package` command compiles, verifies, and builds the project. The resulting compiled code is packaged into a `war` web archive that can be 
+found under the `system/target` directory. The archive contains the application that is needed to run the microservice on an Open Liberty server, and it is 
+now ready to be injected into a Docker container for deployment.
 
 == Containerizing the system microservice
 
-=== Re-using the Docker daemon
+=== Reusing the Docker daemon
 
-To speed up the development process, the built-in Minishift Docker daemon can be re-used to avoid needing to build a Docker registry on the the host machine. 
-To re-use the Docker daemon, run the following command:
+To speed up the development process, the built-in Minishift Docker daemon can be reused to avoid having to build a Docker registry on the host machine. 
```

Comment:
```suggestion
To simplify the local deployment process, the built-in Minishift Docker daemon can be reused to avoid having to build a Docker registry on the host machine. 
```